pytripgui/settings.py

```diff
@@ -84,6 +102,15 @@ def get_value(self, query):
                 return None
         return temp
 
+    def get_value_str(self, query):
+        """ Wrapper which will return empty string if key is not found.
+        """
+        # TODO: better to test if temp is string, and if not return ""
+        temp = self.get_value(query)
```

Comment:
What about using `get` method of dictionary in  `get_value` method ? 
http://stackoverflow.com/questions/11041405/why-dict-getkey-instead-of-dictkey